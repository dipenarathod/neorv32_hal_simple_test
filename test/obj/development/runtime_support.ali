V "GNAT Lib v14"
A -Og
A -gnatA
A --RTS=/home/dipen/.local/share/alire/builds/bare_runtime_14.0.0_095db6f0/282b01b920f0d5bb2bac604ac6d9e811f26d175144bc99af963e0381e797ee94/
A -mabi=ilp32
A -ffunction-sections
A -fdata-sections
A -g
A -gnatwa
A -gnatw.X
A -gnatVa
A -gnaty3
A -gnatya
A -gnatyA
A -gnatyB
A -gnatyb
A -gnatyc
A -gnaty-
A -gnatyd
A -gnatye
A -gnatyf
A -gnatyh
A -gnatyi
A -gnatyI
A -gnatyk
A -gnatyl
A -gnatym
A -gnatyn
A -gnatyO
A -gnatyp
A -gnatyr
A -gnatyS
A -gnatyt
A -gnatyu
A -gnatyx
A -gnatW8
A -misa-spec=20191213
A -march=rv32i_zicsr_zifencei
P ZX

RN
RV NO_IMPLEMENTATION_PRAGMAS

U runtime_support%b	runtime_support.adb	4ff5847b NE OO PK

U runtime_support%s	runtime_support.ads	2d4be45f EE NE OO PK
W interrupts%s		interrupts.adb		interrupts.ali

D interrupts.ads	20250921075146 56bb57a4 interrupts%s
D runtime_support.ads	20250922081033 7bf0b3fb runtime_support%s
D runtime_support.adb	20250922080258 34053780 runtime_support%b
D system.ads		20250921045803 28e4acdd system%s
G a e
G c Z s b [exit_handler runtime_support 4 14 none]
X 1 interrupts.ads
1K9*Interrupts 79e15 2|1w6 1r22
X 2 runtime_support.ads
3K9*Runtime_Support 6l5 6e20 3|4b14 11l5 11t20
4U14*Exit_Handler 3|5b14 10l8 10t20

