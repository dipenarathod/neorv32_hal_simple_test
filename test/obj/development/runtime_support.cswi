20250925145145
-c
-x
ada
-gnatA
--RTS=/home/dipen/.local/share/alire/builds/bare_runtime_14.0.0_095db6f0/282b01b920f0d5bb2bac604ac6d9e811f26d175144bc99af963e0381e797ee94/
-march=rv32i_zicsr_zifencei
-mabi=ilp32
-Og
-ffunction-sections
-fdata-sections
-g
-gnatwa
-gnatw.X
-gnatVa
-gnaty3
-gnatya
-gnatyA
-gnatyB
-gnatyb
-gnatyc
-gnaty-d
-gnatye
-gnatyf
-gnatyh
-gnatyi
-gnatyI
-gnatyk
-gnatyl
-gnatym
-gnatyn
-gnatyO
-gnatyp
-gnatyr
-gnatyS
-gnatyt
-gnatyu
-gnatyx
-gnatW8
