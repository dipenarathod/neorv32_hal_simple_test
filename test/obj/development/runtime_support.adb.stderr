runtime_support.adb:1:04: (style) space required [-gnatyt]
runtime_support.adb:2:04: (style) space required [-gnatyt]
runtime_support.adb:11:21: (style) trailing spaces not permitted [-gnatyb]
runtime_support.ads:1:06: warning: no entities of "Interrupts" are referenced [-gnatwu]
runtime_support.ads:1:18: warning: use clause for package "Interrupts" has no effect [-gnatwu]
